utils/build/virtual_machine/provisions/auto-inject/auto-inject_installer_manual.yml

```diff
@@ -47,9 +51,15 @@
     echo "AGENT VARIABLES CONFIGURED FROM THE SCENARIO:"
     cat scenario_agent.env
     export $(cat scenario_agent.env | xargs)
-  
-    sudo -E sh -c "sudo mkdir -p /etc/datadog-agent && echo \"api_key: ${DD_API_KEY}\" > /etc/datadog-agent/datadog.yaml"
-    DD_REPO_URL=${DD_injection_repo_url} DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+
+    sudo -E sh -c "sudo mkdir -p /etc/datadog-agent && printf \"api_key: ${DD_API_KEY}\nsite: datadoghq.com\n\" > /etc/datadog-agent/datadog.yaml"
+    DD_REPO_URL=${DD_injection_repo_url} bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+    
+    echo "installer stdout"
+    sudo cat /tmp/datadog-installer-stderr.log
+
+    echo "installer stderr"
+    sudo cat /tmp/datadog-installer-stderr.log
```

Comment:
could we copy this files to /var/log/datadog? or download from the machine somehow?
[vm logs](https://github.com/DataDog/system-tests/blob/main/utils/build/virtual_machine/provisions/auto-inject/auto-inject-vm_logs.yml)

Reply:
I couldn't find where this was extracted, I will add it :) 
Thanks